helical/models/geneformer/fine_tuning_model.py

```diff
@@ -0,0 +1,203 @@
+from typing import Literal, Optional
+from helical.models.base_models import HelicalBaseFineTuningHead, HelicalBaseFineTuningModel, HelicalRNAModel
+from helical.models.fine_tune.fine_tuning_heads import ClassificationHead
+from sklearn.metrics import accuracy_score
+import torch
+from torch import optim
+from torch.nn.modules import loss
+from helical.models.geneformer.geneformer_utils import gen_attention_mask, get_model_input_size, pad_tensor_list
+from datasets import Dataset
+import logging
+from tqdm import trange
+from transformers import get_scheduler
+
+
+logger = logging.getLogger(__name__)
+
+class GeneformerFineTuningModel(HelicalBaseFineTuningModel):
```

Comment:
The `forward` function from `HelicalBaseFineTuningModel` is not implemented here